### ISSUE TITLE
No way to set default communication device

### ISSUE THREAD

**Issue by Unluckymichell** [roblatour/setvol#5]
I couldn't find a way to set the Windows default communication device
Unfortunately this is the main reason I need a way to control windows audio as my windows keeps messing up my settings when I plug and unplug devices

Exact steppt taken:

- Go to systemcontroll > sound 
- Set device as standard (both communication and normal)
- Use `setvol makedefault device DEVICENAME` to set as default
- See changes in systemcontroll > sound 

![image](https://user-images.githubusercontent.com/39999330/215093989-bfb1238b-ac86-488c-87a8-e1a7ea283912.png)

![image](https://user-images.githubusercontent.com/39999330/215094194-f172f212-d566-4e76-b0a6-7168483e125f.png)

![image](https://user-images.githubusercontent.com/39999330/215094227-0edbf770-5ef7-4947-8a17-9431fc56cd20.png)

**It is possible to implement an additional command to set the default communication device or modify the setdefault command to set both the communication and the "normal" audio device
It would complete this tool for any use case I can think of**

It's still a really easy-to-use command-line tool though! Windows Audio always bugs me and I've wanted to automate it for ages. Even without this function a very helpful program!

Testet in Version 3.3 of setVol

![image](https://user-images.githubusercontent.com/39999330/215093578-1b9f408b-dcb9-4c9f-b94c-9dae2887af33.png)

_Testet in Microsoft Windows 11 Pro 22H2_

**Comment by roblatour** [comment]
I just took a look, and it appears as this may very well be doable.  Please email me at info@rlatour.com

**Comment by roblatour** [comment]
I took a look and it wasn't too involved of a change; in fact, I just made it, tested it, found it is working fine, and promoted it.

Accordingly, the webpage, code on github, and downloadable executable have now been updated.

As for the webpage ( rlatour.com/setvol ) you may need to refresh your browser (usually F5) or view it using incognito mode as the old version will likely be cached for a while.

In short you now have two commands you can use

setvol makedefault device xxxxx
and
setvol makedefaultcomm device xxxx 
the first works as before, the second will set you default comm device for playing and recording (depending on the device you specify)
also, as before 
setvol device
will list all your devices.

Hope this helps

**Comment by roblatour** [comment]
Just made a minor tweak, now updated (Jan 28th 2023,10:12am EST)

**Comment by Unluckymichell** [comment]
Realy cool! Did not think you would respond that Quick. Thanks, this really now is exactly what I need. I will try out the new version later today, but i think it's fair to close this now.

**Comment by roblatour** [comment]
You're very welcome ...

... and ok, I will close this now as I have already tested it.  

However, when you get a chance to test it yourself please let me know how it worked out for you.  You can e-mail me at the contact e-mail shown near the bottom of the website https://rlatour.com/setvol  (I don't want to include it here in case it gets picked up by e-mail spam bots).
